types/react/index.d.ts

```diff
@@ -1838,7 +1838,7 @@ declare namespace React {
      *
      * @see {@link https://react.dev/reference/react/useDeferredValue}
      */
-    export function useDeferredValue<T>(value: T): T;
+    export function useDeferredValue<T>(value: T, initialValue?: T): T;
```

Comment:
Just need to also update the JSDoc (`@param initialValue A value to use (...)`)

Reply:
c&p what we have in docs:
> A value to use during the initial render of a component. If this option is omitted, useDeferredValue will not defer during the initial render, because there’s no previous version of value that it can render instead.